### ISSUE TITLE
Please allow to browse the SDK source code.

### ISSUE THREAD

**Issue by zapta** [Wiz-IO/wizio-pico#68]
TL;DR, Make the Go To Definition link working for Pico C++ SDK source files.

Let's say that I have a statement like this

  gpio_set_function(PICO_DEFAULT_I2C_SDA_PIN, GPIO_FUNC_I2C);

Following its definition in vscode/platformio using the popup menu

![image](https://user-images.githubusercontent.com/2184769/117695479-1fef5280-b175-11eb-888e-e04e33d254d8.png)

brings the function declaration but not it's implementation:

![image](https://user-images.githubusercontent.com/2184769/117695622-4614f280-b175-11eb-96b4-fde37eb65bb4.png)

Please allow to also jump to definitions of SDK functions. The SDK source files seems to be available since I can step into them in the debugger but they are not show up in the Go To Definitions links.  

Motivation:  looking at the source code of the SDK is very important to understand what certain functions are doing, beyond what is described in the external documentation. I am coming from STM32 and platformio does allow to browse into the HAL and other SDK files.

**Comment by Wiz-IO** [comment]
Hi
this function is controlled from VSCode
PIO have only
![reb](https://user-images.githubusercontent.com/14574834/117697815-974fd100-b1cb-11eb-8681-55feaff65b4a.jpg)

**Comment by Wiz-IO** [comment]
src git is here
https://github.com/Wiz-IO/framework-wizio-pico
and is installed in 
C:\Users\USER_NAME\.platformio\packages\framework-wizio-pico

**Comment by zapta** [comment]
Created this question, let's see what it will bring. 

https://community.platformio.org/t/how-to-make-sdk-source-files-accessible-via-go-to-definition/21347

**Comment by zapta** [comment]
Adding this to build flags in platformio.ini provide Go To Definition access to some SDK functions (e.g. gpio). Not sure if it covers all files.

    -I "$PROJECT_CORE_DIR/packages/framework-wizio-pico/SDK111/hardware"

I think that the trick is to have in .vscode/c_cpp_properties.json (an auto generated file?) the paths of the SDK source files. This will make the files accessible out of the box.

**Comment by Wiz-IO** [comment]
In hardware not have header files
will check and will add in builder

you can test add path here
https://github.com/Wiz-IO/wizio-pico/blob/main/builder/frameworks/common.py#L112
